### PR TITLE
View mcorr shifts

### PR DESCRIPTION
**Pull request by ArjunPutcha**
Handler within caiman_extensions/mcorr module for processing shifts arrays

### CODE REVIEW

File: mesmerize_core/caiman_extensions/mcorr.py [hunk 1]
```diff
@@ -46,9 +46,10 @@ def get_output(self) -> np.ndarray:
         mc_movie = np.reshape(Yr.T, [T] + list(dims), order="F")
         return mc_movie
 
-    def shifts_handler(
-            self, shifts: np.ndarray, pw_rigid: bool=False
-    ):
+    @validate("mcorr")
+    def get_shifts(
+            self, pw_rigid: bool = False
+    ) -> Tuple[List[np.ndarray], List[np.ndarray]]:
```
Comment (kushalkolar) [inline mesmerize_core/caiman_extensions/mcorr.py:52]: I think you can make a more informative docstring and then good to go!

File: mesmerize_core/caiman_extensions/mcorr.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 from ..batch_utils import get_full_data_path
 from .common import validate
+from matplotlib import pyplot as plt
```
[the remark this answers is not in the document]
Comment (kushalkolar) [reply mesmerize_core/caiman_extensions/mcorr.py:9]: @ArjunPutcha

[the remark this answers is not in the document]
Comment (kushalkolar) [reply mesmerize_core/caiman_extensions/mcorr.py:9]: @ArjunPutcha if you remove this I will merge

**Review by kushalkolar — COMMENTED** [review]
just remove the matplotlib import

File: mesmerize_core/caiman_extensions/mcorr.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ArjunPutcha) [reply mesmerize_core/caiman_extensions/mcorr.py]: removed

File: mesmerize_core/caiman_extensions/mcorr.py (hunk 1 above)
Comment (ArjunPutcha) [reply mesmerize_core/caiman_extensions/mcorr.py:51]: Updated. Lmk!

**Pull request merged**